flink-clients/src/main/java/org/apache/flink/client/cli/ProgramOptions.java

```diff
@@ -62,17 +65,63 @@
 
 	private final SavepointRestoreSettings savepointSettings;
 
+	// Flag indicating whether the job is a Python job.
+	private final boolean isPython;
+
 	protected ProgramOptions(CommandLine line) throws CliArgsException {
 		super(line);
 
 		String[] args = line.hasOption(ARGS_OPTION.getOpt()) ?
-				line.getOptionValues(ARGS_OPTION.getOpt()) :
-				line.getArgs();
+			line.getOptionValues(ARGS_OPTION.getOpt()) :
+			line.getArgs();
+
+		isPython = line.hasOption(PY_OPTION.getOpt()) | line.hasOption(PYMODULE_OPTION.getOpt());
+		// If specified the option -py(--python)
+		if (line.hasOption(PY_OPTION.getOpt())) {
+			// Cannot use option -py and -pym simultaneously.
+			if (line.hasOption(PYMODULE_OPTION.getOpt())) {
+				throw new CliArgsException("Cannot use option -py and -pym simultaneously.");
+			}
+			// The args format is : py ${python.py} [optional]pyfs [optional]${py-files} [optional]other args.
```

Comment:
`[optional]pyfs` -> `[optional] pyfs`? add a space.
`py ${python.py}` -> `-py ${python.py}`?
`pyfs [optional]${py-files}` -> `-pyfs [optional]${py-files}`?

Reply:
I have added more decription about the transformation betweet CLI args and PythonDriver args in the code note.